ports/stm32/boards/NUCLEO_H563ZI/mpconfigboard.h

```diff
@@ -0,0 +1,80 @@
+#define MICROPY_HW_BOARD_NAME               "NUCLEO_H563ZI"
+#define MICROPY_HW_MCU_NAME                 "STM32H563ZI"
+
+#define MICROPY_PY_PYB_LEGACY               (0)
+#define MICROPY_HW_ENABLE_INTERNAL_FLASH_STORAGE (1)
+#define MICROPY_HW_ENABLE_RTC               (1)
+#define MICROPY_HW_ENABLE_RNG               (1)
+#define MICROPY_HW_ENABLE_ADC               (1)
+#define MICROPY_HW_ENABLE_DAC               (0) // TODO: requires DMA, which is not yet ready
```

Comment:
This can be enabled now (DAC/DMA support was merged).

Reply:
Done